README.md

```diff
@@ -46,9 +46,9 @@ workshop repo to teach you about testing JavaScript applications.
 
 ## Topics covered
 
-1. Unit Testing with [Jest](http://facebook.github.io/jest)
-2. Integration Testing with [Jest](http://facebook.github.io/jest)
-3. End to End (E2E) Testing with [Cypress](https://www.cypress.io/)
+1.  Unit Testing with [Jest](http://facebook.github.io/jest)
+2.  Integration Testing with [Jest](http://facebook.github.io/jest)
+3.  End to End (E2E) Testing with [Cypress](https://www.cypress.io/)
```

Comment:
[This isn't the first time](https://github.com/paypal/downshift/pull/370#discussion_r173705098) I've run into this sort of "automatic re-spacing" changes incurred when running the `all-contributors` script.